packages/sdk/src/lend/bindings/verbsLendBinding.ts

```diff
@@ -0,0 +1,32 @@
+import type { LendProvider } from '@/lend/provider.js'
+import type { LendReadOperations } from '@/types/lend.js'
+
+/**
+ * Bind a lend provider to the Verbs instance with read-only operations
+ * @param provider - The lend provider instance
+ * @returns LendReadOperations interface with bound methods
+ */
+export function bindLendProviderToVerbs(
```

Comment:
these `bindLendProviderToVerbs()` and `bindLendProviderToWallet()` functions are a possible alternative to creating `LendProviderVerbsNamespace` and `LendProviderWalletNamespace` classes, though I'm not really sold on this design either tbh. ideas @tremarkley?